src/index.css

```diff
@@ -8,5 +8,51 @@
   font-family: 'Work Sans', sans-serif;
 }
 
+/* Global Styles */ 
 
+/* Ensures a parent element with this class name stretches to the height of floated children */
+.clearfix:after {
+  content: "";
+  display: block;
+  clear: both;
+}
+
+.container {
+  max-width: 1280px;
+  margin: 0 auto;
+}
+
+.primarySection {
+  padding: 120px 0;
+}
```

Comment:
I gathered that you were doing a section-by-section sort of thing, which is lovely and standard. To create a sense of vertical rhythm, you'll want consistent white space between those sections. This class can be thrown on any new section to give it a consistent top and bottom padding.